docs/miscellaneous/logging.rst

```diff
@@ -0,0 +1,29 @@
+.. _logging_section:
+
+Logging
+=======
+
+In **alchemlyb**, we use :mod:`loguru` for logging. By default, the
+:mod:`loguru` will print the logging information into the `stderr`.
+
```

Comment:
Add a link to https://loguru.readthedocs.io/en/stable/overview.html#entirely-compatible-with-standard-logging along the lines of
```reST
The loguru logger is compatible with the :mod:`logging` module of the Python
standard library and can easily be `configured to log to a *logging* handler<logguru-standard-logging>`_. 

.. _logguru-standard-logging:
    https://loguru.readthedocs.io/en/stable/overview.html#entirely-compatible-with-standard-logging
```